docs/source/system_reference_guide/jobs_maappy.ipynb

```diff
@@ -135,6 +135,61 @@
     ".. note::\n",
     "This method submits a request to cancel a job so it may take several minutes before the cancellation takes effect. Users may check the job's status to confim cancellation. Cancelled jobs will return a status of `Dismissed` if they were cancelled after starting. Queued jobs that are cancelled will be purged from the system."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7100c385",
+   "metadata": {},
+   "source": [
+    "## List Jobs\n",
+    "\n",
+    "1. Use the `listJobs` method, specifying a username, to retrieve a list of jobs that user has submitted.\n",
```

Comment:
is there a listJobs() that gets everything? why not have listJobs(username=“mlucas”) as syntax?

Reply:
`listJobs(username="mlucas")` is valid. I did not want to remove `listJobs("mlucas")` because users are currently using the latter. If we want to clean maap-py up (which I believe there are plans to) then we can revisit this and make sure to add deprecation warnings for any of the interfaces we change.